Source/WebCore/Modules/mediastream/RTCStatsReport.h

```diff
@@ -397,7 +397,7 @@ class RTCStatsReport : public RefCounted<RTCStatsReport> {
         CodecStats(const GstStructure*);
 #endif
 
-        std::optional<uint32_t> payloadType;
+        uint32_t payloadType;
```

Comment:
```suggestion
        uint32_t payloadType { 0 };
```
Our binding code receives optional<uint32_t>, II would expect these optional to contain values but still...

Reply:
Will rebase and update with suggested change and then will merge it. Thanks for your review. :-)

Reply:
Done.